mirrord/analytics/src/lib.rs

```diff
@@ -29,6 +29,45 @@ pub enum AnalyticsError {
     Unknown,
 }
 
+#[derive(Default, Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecutionKind {
+    Container,
+    #[default]
```

Comment:
we don't send strings in the analytics on purpose (see target type) - set it to an int enum (0 for exec, 1 for container and so on)

Reply:
I can make it bitflags so we can do extension + exec or (extension + contianer in the future)

Reply:
I don't think we need, we can make "extension" a flag on it's own then specify which kind of extension.

Reply:
Ok did it simple